modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -282,6 +282,58 @@ Opening a non-default collection on an memcached bucket now fails fast.
 * https://issues.couchbase.com/browse/JVMCBC-983[JVMCBC-983]:
 Ignore slow subscribers on certain Flux intervals.
 
+== Version 1.1.8 (1 March 2022)
+https://docs.couchbase.com/sdk-api/couchbase-scala-client-1.1.8/com/couchbase/client/scala/index.html[API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**scala-client:1.1.8**
+* com.couchbase.client:**core-io:2.1.8**
+* io.projectreactor:**reactor-core:3.4.15**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+== Bug Fixes
+
+* https://issues.couchbase.com/browse/JVMCBC-1067[JVMCBC-1067]:
+Internal and external maintenance dependencies are updated to their latest available bugfix releases (including Netty to 4.1.74.Final).
+* https://issues.couchbase.com/browse/JVMCBC-1046[JVMCBC-1046]:
+Added fix to not load the global config if a node is not in the seed node list anymore.
+* https://issues.couchbase.com/browse/JVMCBC-1006[JVMCBC-1006]:
+`ErrorContext` is now included in the message of a `CouchbaseException`.
+
+== Version 1.1.7 (11 August 2021)
+https://docs.couchbase.com/sdk-api/couchbase-scala-client-1.1.7/com/couchbase/client/scala/index.html[API Reference]
```

Comment:
Hmm, looks like this old release API link doesn't work?
Is this correct @daschl?